sagemaker-pipelines/data-ingestion/feature-store-data-ingestion-via-sagemaker-model-building-pipeline.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>Nit: Why not all in one cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>we could, but visually, its the same tho</p>